plugins/aea-crypto-cosmos/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
+                           Version 2.0, January 2004
```

Comment:
Can we not point to root license from Manifest file?

Reply:
yes

Reply:
sorry, no:

https://github.com/pypa/setuptools/issues/2339#issuecomment-683465763

Reply:
but we could add a check that make sure the LICENSE is the same

Reply:
Ok, let's worry about these checks separately (new PR)...there are a few more

Reply:
done in 8af620f